examples/runtime-interaction/README.md

```diff
@@ -0,0 +1,22 @@
+# Runtime interaction
+
+This example shows how we can easily send transactions to a blockchain, just like with `subxt` or a similar client.
+
+This way we can leverage `drink!` to a library for a full-scope runtime simulation engine.
+
+## Running
+
+```bash
+cargo test --release
+```
+
+## `drink::Sandbox` vs `drink::Session`
```

Comment:
Could you please elaborate a bit more on when users might choose one over the other. What are the key differences?

Reply:
added some explanation